services/galley/galley.integration.yaml

```diff
@@ -83,15 +83,6 @@ settings:
           verificationExpiration: 86400
           acmeDiscoveryUrl: null
         lockStatus: unlocked
-    mlsMigration:
-      defaults:
-        status: enabled
-        config:
-          startTime: "2029-05-16T10:11:12.123Z"
-          finaliseRegardlessAfter: "2029-10-17T00:00:00.000Z"
-          usersThreshold: 100
-          clientsThreshold: 50
-        lockStatus: locked
```

Comment:
What's the reason to remove this?

Reply:
Ah this file wasn't supposed to be committed. Thanks for catching it!